tests/test_bs.py

```diff
@@ -369,7 +374,7 @@ def test_bootstrap(set_namespace, setup_bootstrap):
 def test_client(set_namespace, setup_clients, save_log_on_exit):
```

Comment:
I don't get why set_namespace is called here and not in the same way session_id is set. (like in setup_bootstrap)